build.gradle

```diff
@@ -98,7 +98,7 @@ dependencies {
         dropwizardVersion = '3.1.2'
     }
 
-    pgsql 'org.postgresql:postgresql:9.4.1207'
+    pgsql 'org.postgresql:postgresql:42.1.1'
```

Comment:
@adyach I'm not sure I understand your first question. For the versioning, yes it is strange, but here is the documentation that explains it (I also included it in the commit message): https://jdbc.postgresql.org/documentation/faq.html#versioning